### ISSUE TITLE
Dark Souls III (374320) crash on starting new game

### ISSUE THREAD

**Issue by hhyyrylainen** [ValveSoftware/Proton#2537]
# Compatibility Report
- Name of the game with compatibility issues: Dark Souls III
- Steam AppID of the game: 374320

## System Information
- GPU: GTX 1080 Ti
- Driver/LLVM version: nvidia 418.56
- Kernel version: 5.0.6-200
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/hhyyrylainen/b37480d424240a4c72a0d1a2a3fbd1a2
- Proton version: 3.16-4 (recommended), same issue also with 4.2-2

## I confirm:
- [x] that I haven't found an existing compatibility report for this game. I found a workaround from the following issue but @kisak-valve asked me to open a new issue
  - https://github.com/ValveSoftware/Proton/issues/478
- [x] that I have checked whether there are updates for my system available.

https://gist.github.com/hhyyrylainen/8801236fe770a3ea38f6da72959e5a01

## Symptoms 
Game crashes when starting a new game. After clicking "new game" from the main menu the game shows a black screen for a second and then crashes.

## Workaround
@UndeadKernel found a workaround: https://github.com/ValveSoftware/Proton/issues/478#issuecomment-420053552
Basically just rename the video files to prevent the game from trying to play them. With that change I can start a new game just fine.

**Comment by aeikum** [comment]
Something is broken with your OpenAL configuration:

```
AL lib: (WW) GetSymbol: Failed to load jack_error_callback: /lib64/libjack.so.0: undefined symbol: jack_error_callback
AL lib: (WW) jack_msg_handler: Cannot connect to server socket err = No such file or directory
AL lib: (WW) jack_msg_handler: Cannot connect to server request channel
AL lib: (WW) jack_msg_handler: jack server is not running or cannot be started
AL lib: (WW) jack_msg_handler: JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
AL lib: (WW) jack_msg_handler: JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
AL lib: (WW) ALCjackBackendFactory_init: jack_client_open() failed, 0x11
AL lib: (WW) alc_initconfig: Failed to initialize backend "jack"
```

I suspect fixing that will fix this problem. However, I don't think we use openal in 3.16 or 4.2. Are you sure you're using one of those versions? If you actually upgrade to 3.16 or 4.2, then I think that may fix this, too.

**Comment by hhyyrylainen** [comment]
I am playing through Steam. It says it's using "Proton 3.16-4" on the game's page. And I tried the newer version through the "properties" force specific proton version option. So the only thing that I can think of causing it to use something else is that I have tried switching between versions and it is somehow "stuck" on an older version even though it says it's using another version.

As to OpenAL I haven't noticed any problems anywhere else other than this game.

**Comment by deathxxx123** [comment]
> # Compatibility Report
> 
>     * Name of the game with compatibility issues: Dark Souls III
> 
>     * Steam AppID of the game: 374320
> 
> 
> ## System Information
> 
>     * GPU: GTX 1080 Ti
> 
>     * Driver/LLVM version: nvidia 418.56
> 
>     * Kernel version: 5.0.6-200
> 
>     * Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/hhyyrylainen/b37480d424240a4c72a0d1a2a3fbd1a2
> 
>     * Proton version: 3.16-4 (recommended), same issue also with 4.2-2
> 
> 
> ## I confirm:
> 
>     * [x]  that I haven't found an existing compatibility report for this game. I found a workaround from the following issue but @kisak-valve asked me to open a new issue
>       
>       * #478
> 
>     * [x]  that I have checked whether there are updates for my system available.
> 
> 
> https://gist.github.com/hhyyrylainen/8801236fe770a3ea38f6da72959e5a01
> ## Symptoms
> 
> Game crashes when starting a new game. After clicking "new game" from the main menu the game shows a black screen for a second and then crashes.
> ## Workaround
> 
> @UndeadKernel found a workaround: [#478 (comment)](https://github.com/ValveSoftware/Proton/issues/478#issuecomment-420053552)
> Basically just rename the video files to prevent the game from trying to play them. With that change I can start a new game just fine.

This is because sound configuration. For me, i can play with 2.1 sound(this one have speak from clips and all sounds as well). With 5.1 sound and Proton 3.16-4, game exit exactly like you. With Proton 4.2-3 i can play with 5.1 sound, but with no speak from clips(all other sounds in-game seems working fine). Game do not crash.
